ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -259,6 +259,9 @@ then
 
 	# Possible external compasses
 	hmc5883 -C -T -X start
+
```

Comment:
Given [The default](https://github.com/PX4/Firmware/blob/master/src/drivers/lis3mdl/lis3mdl.cpp#L1648) 
Would the first invocation on line 257 already start both?
>\# Internal SPI bus
> 	lis3mdl -R 0 start

If not should we add the explicit -S  to the first?